cdisc_rules_engine/dataset_builders/contents_define_vlm_dataset_builder.py

```diff
@@ -22,6 +22,7 @@ def build(self):
         "define_vlm_label",
         "define_vlm_data_type",
         "define_vlm_role",
+        "define_vlm_mandatory",
```

Comment:
`define_vlm_mandatory` and `define_variable_mandatory` also need to be added to the `schema.MetaVariables` files

Reply:
This has not been addressed yet

Reply:
Added both to the schema.MetaVariables file

Reply:
I checked and added to the MetaVariables.